client/rest-high-level/src/main/java/org/elasticsearch/client/IndicesClient.java

```diff
@@ -186,6 +188,35 @@ public void getMappingsAsync(GetMappingsRequest getMappingsRequest, RequestOptio
             GetMappingsResponse::fromXContent, listener, emptySet());
     }
 
+    /**
+     * Retrieves the field mappings on an index or indices using the Get Field Mapping API.
+     * See <a href="https://www.elastic.co/guide/en/elasticsearch/reference/current/indices-get-field-mapping.html">
+     * Get Field Mapping API on elastic.co</a>
+     * @param getFieldMappingsRequest the request
+     * @param options the request options (e.g. headers), use {@link RequestOptions#DEFAULT} if nothing needs to be customized
+     * @return the response
+     * @throws IOException in case there is a problem sending the request or parsing back the response
+     */
+    public GetFieldMappingsResponse getFieldMappings(GetFieldMappingsRequest getFieldMappingsRequest,
```

Comment:
it should be singular getFieldMapping. that's what we have in the SPEC

Reply:
there is inconsistency with a class name `GetFieldMappingsResponse`

Reply:
i see the difficulties - sounds ok to me